lib/onboarding_tour.dart

```diff
@@ -0,0 +1,190 @@
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:onboarding/onboarding.dart';
+
+class OnboardingTour extends StatefulWidget {
+  static const route = "/" + subRouteName;
+  static const subRouteName = "onboarding";
+
+  const OnboardingTour({Key? key}) : super(key: key);
+
+  @override
+  State<OnboardingTour> createState() => _OnboardingTourState();
+}
+
+class _OnboardingTourState extends State<OnboardingTour> {
+  late ElevatedButton materialButton;
+  late int index;
+
+  final onboardingPagesList = [
+    PageModel(
+      widget: SingleChildScrollView(
+        controller: ScrollController(),
+        child: OnboardingPage(
+            title: "WELCOME TO 10101",
+            description:
+                "The one-stop Bitcoin app with non-custodial trading over Lightning. 10101 is all about keeping your keys where they belong: with you.",
```

Comment:
nit: It would be nice to suggest to the user that it's TenTenOne to clear any ambiguity

```suggestion
                "The one-stop Bitcoin app with non-custodial trading over Lightning. 10101 (TenTenOne) is all about keeping your keys where they belong: with you.",
```

Reply:
I can write it as a word on the first screen, but I am not convinced that the onboarding has to mention how to pronounce the name 😅

Reply:
to me this is part of onboarding - there's no other spot to teach the users :P